src/OpenTelemetry.Instrumentation.StackExchangeRedis/StackExchangeRedisCallsInstrumentationOptions.cs

```diff
@@ -21,5 +23,9 @@ namespace OpenTelemetry.Instrumentation.StackExchangeRedis
     /// </summary>
     public class StackExchangeRedisCallsInstrumentationOptions
     {
+        /// <summary>
+        /// Gets or sets the maximum time that should elapse between flushing the internal buffer of Redis profiling sessions and creating <see cref="Activity"/> objects. Default value: 00:00:10.
+        /// </summary>
+        public TimeSpan MaxFlushInterval { get; set; } = TimeSpan.FromSeconds(10);
```

Comment:
@cijothomas One thing to note here. Before the background thread pulled every 1 second, which was not configurable. This change makes it configurable and sets the time at 10 seconds by default.

Reply:
Yes this sounds reasonable for me.

Reply:
Do we need MaxFlushInternal? or just `FlushInterval` is fine?

Reply:
I think FlushInterval is fine. i will change soon